vbfml/scripts/err_analysis

```diff
@@ -196,6 +225,10 @@ def predict(
 
     df_non_feature = pd.concat(high_level_dfs, ignore_index=True)
 
+    assert len(predictions) == len(
+        df_non_feature
+    ), f"prediction and df_non_feature are not matching ({len(predictions)}-{len(df_non_feature)})"
```

Comment:
Good check. Could you alter the error message slightly, saying something like:
```
f"Length of predictions and df_non_feature are not matching ({len(predictions)}-{len(df_non_feature)})"
```

Reply:
corrected